endoreg_db/models/media/video/video_file_meta.py

```
@@ -0,0 +1,363 @@
+        # Update state
+        state = video.get_or_create_state() # Use State helper
+        state.sensitive_data_retrieved = True
+        state.save(update_fields=["sensitive_data_retrieved"])
+        logger.info("Successfully updated/created SensitiveMeta and set state for video %s.", video.uuid)
+
+        # Save the VideoFile instance itself if requested and if sensitive_meta was linked
+        if save_instance:
+            update_fields = ["sensitive_meta"]
+            # Check if derived fields also need updating
+            if video.sensitive_meta:
+                if not video.patient and video.sensitive_meta.pseudo_patient: update_fields.append("patient")
```

Ruff (0.8.2)

Multiple statements on one line (colon)

(E701)